### ISSUE TITLE
Aventurische Tiergefährten – Kreaturen und Ausbildung (ATI)

### ISSUE THREAD

**Issue by nilsriesenberg** [elyukai/optolith-client#549]
**Aventurische Tiergefährten**
>
> - [ ] S. 8-9 (Aufbau der Tierbeschreibung)
> - [ ] S. 9-28 (Hunde)
> - [ ] S. 29-36 (Katzen)
> - [ ] S. 37-39 (Esel & Eselartige)
> - [ ] S. 40-50 (Pferde)
> - [ ] S. 51-55 (Ponys)
> - [ ] S. 56-61 (Hoftiere)
> - [ ] S. 62-68 (Boten-, Jagd- und Ziervögel)
> - [ ] S. 69-91 (Besondere Begleiter)

**Anhang**
>
> - [ ] S. 92 (Körperkraft schwacher Wesen)
> - [ ] S. 92-93 (Gangarten, Erschöpfung und Geschwindigkeit)
> - [ ] S. 93-97 (Kosten, Futter, Versorgung, Zubehör)
> - [ ] S. 97-98 (Tierzucht)
> - [ ] S. 98-105 (Abrichten und Dressieren)
> - [ ] S. 106 (Tierheilkunde)
> - [ ] S. 106-109 (Tierkrankheiten)
> - [ ] S. 109-111 (Heilige Tiere)
> - [ ] S. 111-116 (Vertrautentiere)
> - [ ] S. 117 (Tiervor- und -nachteile)
> - [ ] S. 117-120 (Vorteile)
> - [ ] S. 120-122 (Nachteile)
> - [ ] S. 123-125 (Tiersonderfertigkeiten)
> - [ ] S. 125-127 (Kampfsonderfertigkeiten für Tiere)
> - [ ] S. 127-128 (Kampf in drei Dimensionen)
> - [ ] S. 128 (Neue Sonderfertigkeiten für Kulturschaffende)

**Tables**
>
> - [ ] L10n de-DE

**Comment by mwberlindeu** [comment]
Ergänzung:
Version 1.40 ermöglicht momentan nicht vollständig einen Tiergefährten anzulegen. Ich habe es mit einem Pferd versucht, aber z.B. können nicht mehrere Angriffsvarianten (Tritt, Niederreiten, etc.) eingegeben werden, sondern leider nur immer eine. Die Aufnahme der Publikation wäre praktisch und geht in der Implementierung so weit ich es einschätze, über die Ergänzung statischer Daten hinaus.

**Comment by elyukai** [comment]
Ja, für die Publikation muss mehr passieren als nur Daten einzutragen. Ein Schema für Kreaturen existiert aktuell nicht einmal und eine vernünftige Verwaltung für deren Werte (auch im Sinne von Vertrauten) ebenso wenig, was der Grund ist, weshalb das Regelwerk aktuell nicht drin ist – es macht ohne eine erweiterte Verwaltung einfach keinen Sinn. Das wäre dementsprechend ein Teil der Implementierung hier.
